src/ui/include/EditorNS/editor.h

```diff
@@ -230,6 +230,7 @@ namespace EditorNS
         Q_INVOKABLE void setEOLVisible(const bool showeol);
         Q_INVOKABLE void setWhitespaceVisible(const bool showspace);
         Q_INVOKABLE void setMathEnabled(const bool enabled);
+        Q_INVOKABLE void setReadOnly(const bool readOnly);
```

Comment:
Could you also add a `bool isReadOnly() const`?

Reply:
I'll do. I'm working on improvements this feature and I already had need to use `isReadOnly` :)